src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1630,6 +1642,8 @@ public function getEncodings(): array
 
     /**
      * Gets a list of content types acceptable by the client browser in preferable order.
+     *
+     * @return string[] List of content types in preferable order
```

Comment:
I would remove all return descriptions as they just say the same as the main description.